cypress/e2e/search_spec.cy.js

```diff
@@ -0,0 +1,66 @@
+describe('Search Component', () => {
+  beforeEach(() => {
+    cy.intercept("GET", "https://streeteatz-be-b15261620498.herokuapp.com/vendors/", {
+      statusCode: 200,
+      fixture: "truckData.json"})
+      .visit("http://localhost:3000/")
+  })
+  it('Should Render a search bar', () => {
+   cy.get('.search-field').should('exist')
+  })
+  it("Should fill in the search bar", () => {
+    cy.get('.search-field').type('noodles')
+      .should('have.value', 'noodles')
+  })
+  it("Should hit the search icon and show the results", () => {
+    cy.get('.search-field').type('noodles')
+    cy.get('.searchQuerySubmit').click()
+    cy.get('.name').contains('p', "Mac and Noodles")
+    cy.get('.location').contains('p', '500 E 17th Ave Denver, CO')
+    cy.get('.upvotes').contains('p', '666')
+    cy.get('.downvotes').contains('p', '12')
+  })
+  it('Should be able to find the favorite button', () => {
+    cy.get(`[value="favorites"]`).should('exist')
+  })
+  it('Should have no favorites by default', () => {
+    cy.get(`[value="favorites"]`).click()
+    cy.get(`.card-container`).should('not.exist')
+  })
+  it("Should be able to favorite a card", () => {
```

Comment:
I still haven't used nth-child, i'll have to ask you about this. But looks great!

Reply:
Ya so on the cypress page if you click the button on the top left that looks like a target, you can use it to click and view the exact term you need to target anything

Reply:
I have not tried that technique yet either

Reply:
Its the method if you guys want to hop on zoom rq i can show you, itll take like a minute